### ISSUE TITLE
AC sync?

### ISSUE THREAD

**Issue by yairigal** [nryhbhue/Electra-AC-Remote#2]
how can i turn the ac ON/OFF if i dont know the AC current state?
is there an option to get the data from the AC to sync with the RP/UNO?

**Comment by oriharel** [comment]
@yairigal were you able to make this work?

**Comment by yairigal** [comment]
> @yairigal were you able to make this work?

No, I just kept the current state of the AC synced with my data, in case someone change the state with the remote i just have to manually overwrite it.

**Comment by oriharel** [comment]
@yairigal I actually meant this entire project. I ran it in my Pi and nothing happened.

**Comment by dov** [comment]
I also tried the codes and couldn't get them to work. I outlined my experience in more detail in #3.

**Comment by yairigal** [comment]
@oriharel.
Yea, Its actually still running on my arduino (esp 8266).
I'm not sure what causes the problem, the codes you generate or the hardware IR stuff.

I don't remember what i did back then, but you can look at my repo for the code.
https://github.com/yairigal/Arduino_AControl/blob/master/yAcRemote.h

**Comment by HWiese1980** [comment]
In case this is still relevant... I managed to get things working with my AEG AC (which is apparently compatible with Electra ACs) using ESPHome with the IRremoteESP8266 library and code inspired by [Hagakurje/ESPHome_Gree_AC](https://github.com/Hagakurje/ESPHome_Gree_AC). You can see how I did it in my repository [HWiese1980/electra_chillflex_esphome](https://github.com/HWiese1980/electra_chillflex_esphome). 

One-way of course... afaik the AC has no back channel (yet; I'm out of warranty and thinking of integrating a D1 mini for better control :-D )
